sensor-system/migrations/v0.2.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE Location (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(32) NOT NULL,
+    address VARCHAR(128) NOT NULL
+);
```

Comment:
```suggestion
CREATE TABLE Location (
    "id" SERIAL PRIMARY KEY,
    "name" VARCHAR(32) NOT NULL,
    "address" VARCHAR(128) NOT NULL,
    "created" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
    "updated" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
    "deleted" TIMESTAMP WITH TIME ZONE,
);
```

Name and address are both keywords in postgres, it shouldn't make a difference but they should be enclosed in double quotes. For consistency, enclose all column names in double quotes.

For our purposes having a single address field is fine but typically you'd want to break it out to address, city, state, zip, country.

Add created, updated, and deleted columns to all the tables except Count.